test/parallel/xyk-pallet.API.poolLiqidity.claimReward.test.ts

```diff
@@ -61,20 +67,21 @@ beforeAll(async () => {
 
   await setupApi();
 
-  [token1, token2] = await Assets.setupUserWithCurrencies(
+  [token1, token2, token3] = await Assets.setupUserWithCurrencies(
```

Comment:
plz, review that token2 and token3 are used because I think they are not.

Reply:
Yes, they used
<img width="453" alt="image" src="https://github.com/mangata-finance/mangata-e2e/assets/105198474/12fb2773-50bb-480e-8f59-488f2a3c81ba">
<img width="429" alt="image" src="https://github.com/mangata-finance/mangata-e2e/assets/105198474/fed30cac-8d67-4032-b2d4-490e84186025">
<img width="429" alt="image" src="https://github.com/mangata-finance/mangata-e2e/assets/105198474/caf6e049-2686-4662-9fbe-9f87ea562d12">